pkg/order/mempool/mempool_impl.go

```diff
@@ -206,8 +206,9 @@ func (mpi *mempoolImpl) generateBlock() (*raftproto.RequestBatch, error) {
 	mpi.batchSeqNo++
 	batchSeqNo := mpi.batchSeqNo
 	batch := &raftproto.RequestBatch{
-		TxList: &pb.Transactions{Transactions: txList},
-		Height: batchSeqNo,
+		TxList:    &pb.Transactions{Transactions: txList},
+		Height:    batchSeqNo,
+		Timestamp: time.Now().UnixNano(),
```

Comment:
The timestamp field in blockHeader is not updated.

Reply:
add the batch's timestamp for the blockHeader.